### DISCUSSION TITLE
/tmp/.X11-unix not owned by root or us

### DISCUSSION THREAD

**Discussion by IcyAlmond** [netblue30/firejail#5425] [Q&A] [answered]
I'm running a programming and I already have 
`noblacklist /tmp/.X11-unix`
but it still says: `/tmp/.X11-unix not owned by root or us`
and if I use zsh in the profile, it tells me that it is owned by "nobody"
How can i fix this?

Edit: Answered and solved by rusty-snake

**Comment by rusty-snake** [comment]
Nobody can help you if you give no details. Which program, if you keep the program secret it is hard to help you. Which profile is used? Firejail version, distro, ...? Who ones `/tmp/.X11-unix` outside the sandbox? root? Then it is clear that `noroot` will change this. What permissions are shown?

**Comment by kmk3** [comment, marked as answer]
Hello, this issue is missing basic debugging information.

Please follow the "Bug report" template in a new issue:

* <https://github.com/netblue30/firejail/issues/new/choose>
